src/Azure.Functions.Cli/Diagnostics/LoggingFilterHelper.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Diagnostics;
+using Microsoft.Azure.WebJobs.Script.Eventing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Functions.Cli
+{
+    public class LoggingFilterHelper
```

Comment:
Could this be internal?

Reply:
Tried this. This needs to be public for Startup and ColoredConsoleLogger.